src/objective-c/tests/InteropTests.m

```diff
@@ -341,6 +341,90 @@ - (void)testLargeUnaryRPCWithV2API {
   [self waitForExpectationsWithTimeout:TEST_TIMEOUT handler:nil];
 }
 
+- (void)testConcurrentRPCsWithErrorsWithV2API {
```

Comment:
As discussed offline, this test is a valid interop test case, but not really putting much load on the system. Maybe have a test like this but adds more load, run longer, and run separately from the interop test suite.

Reply:
I'll add a new stress test job for long running tests as part of a new PR. The new job will run periodically (not on every PR).